curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md

```diff
@@ -26,7 +26,7 @@ Implement a function that takes a string of four digits as its argument, with ea
 | Example input | Example output |
 | --- | --- |
 | <code>solve24("4878");</code> | <code>(7-8/8)*4</code> |
-| <code>solve24("1234");</code> | <code>3*1*4*2</code> |
+| <code>solve24("1234");</code> | <code>3 * 1 * 4 * 2</code> |
```

Comment:
@obbap1 I don't think this is necessary.  Also, if you are going to change one test `text`, you should really changed them all.

Reply:
Got it.  In that case, let's change the table entry to be:
```md
| <code>solve24("1234");</code> | <code>3*1*4*2</code> |
```
and remove the spaces in your change.

Reply:
With that, it looks like this
![image](https://user-images.githubusercontent.com/26006996/73001967-92fb7b00-3e03-11ea-95d9-8b9349445963.png)

You can try it directly as well

Reply:
It has to do with the way that markdown is display using asterisks.  

Try the following and let me know.
```
| <code>solve24("1234");</code> | <code>3&ast;1&ast;4&ast;2</code> |
```

Reply:
Okay, it works fine now.